src/platform/graphics/webgpu/webgpu-graphics-device.js

```diff
@@ -411,8 +411,8 @@ class WebgpuGraphicsDevice extends GraphicsDevice {
         WebgpuDebug.memory(this);
         WebgpuDebug.validate(this);
 
-        // current frame color output buffer
-        const outColorBuffer = this.gpuContext.getCurrentTexture();
+        // current frame color output buffer (fallback to external backbuffer if not available)
+        const outColorBuffer = this.gpuContext?.getCurrentTexture?.() ?? this.externalBackbuffer?.impl;
```

Comment:
should it be `this.externalBackbuffer?.impl.gpuTexture`?

Reply:
Yes! Thanks